.travis.yml

```diff
@@ -14,10 +14,11 @@ cache:
 # https://docs.travis-ci.com/user/languages/java/#projects-using-maven
 #
 # We also want to avoid Maven's install phase to prevent from writing to ~/.m2/repository, which is cached.
-install: mvn dependency:go-offline -Pjacoco,sonar --show-version
+install: mvn dependency:go-offline -Pcode-coverage,sonar --show-version
```

Comment:
Is it using the existing code-coverage profile from kie-parent? Does it work with SonarCloud well? The profile was added in the past, before any integration with SonarCloud.

Reply:
Yes, this is `code-coverage` profile from `kie-parent`. It seems to work as expected.

Reply:
I tried with a new SonarCloud project after I figured out that I need to change project key. The metrics seemed OK (~80% coverage). I'm pretty confident, we'll get the expected report after merging.

Note: the test inclusion filter seems now to work as expected (tests' coverage is not measured). Overall, sonar plugin behavior changed a lot after inheriting from `kie-parent`. I can show you offline.

Reply:
After quickly looking on the configuration in kie-parent, there is no reason why is should not work for unit tests. Just be aware, that if you migrate the build to jenkins, the profile name triggered in jenkins jobs is different.